sdk-api-src/content/windows.graphics.effects.interop/index.md

```diff
@@ -16,7 +16,8 @@ f1_keywords:
 
 ## -description
 
-This header is used by Windows Runtime C++ reference. For more information, see:
+This header is used by multiple technologies. For more information, see:
 
+- [w_graph_fx](../_w_graph_fx/index.md)
```

Comment:
A couple technology areas (5 of them) don't have a "displayName":
* winlocation_com_ref
* winsensors_com_ref
* wlbsprov
* w_graph_fx
* w_ui_comp

Reply:
@tannergooding Thanks for calling these out. Some of these I have found names for, others I think I will remove. I'll go ahead and make these changes in our internal client repo so I can verify the fixes internally before merging the changes, but thanks again for identifying these.